opensrp-anc/src/main/java/org/smartregister/anc/library/util/JsonFormUtils.java

```diff
@@ -595,12 +604,11 @@ public static void startFormForEdit(Activity context, int jsonFormActivityReques
                     (Event) new Event().withBaseEntityId(entityId) //should be different for main and subform
                             .withEventDate(encounterDate).withEventType(ConstantsUtils.EventTypeUtils.UPDATE_REGISTRATION)
                             .withLocationId(encounterLocation).withProviderId(providerId)
-                            .withEntityType(DBConstantsUtils.WOMAN_TABLE_NAME).withFormSubmissionId(org.smartregister.anc.library.util.JsonFormUtils.generateRandomUUIDString())
+                            .withEntityType(DBConstantsUtils.DEMOGRAPHIC_TABLE_NAME).withFormSubmissionId(org.smartregister.anc.library.util.JsonFormUtils.generateRandomUUIDString())
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.anc.library.util.JsonFormUtils.generateRandomUUIDString' due to existing same package import 'org.smartregister.anc.library.util.*'](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=5023982)